bin/pybsn-repl

```diff
@@ -126,6 +128,6 @@ user_ns = dict(ctrl=ctrl, root=ctrl.root, show_schema=show_schema)
 app = TerminalIPythonApp(config=config)
 app.initialize(argv=[])
 app.shell.push(user_ns, interactive=False)
-app.shell.input_transformer_manager.python_line_transforms.append(LineTransformer())
+app.shell.input_transformers_cleanup.append(line_transform)
```

Comment:
The old API was deprecated and removed in `ipython 7` (see https://ipython.readthedocs.io/en/latest/api/generated/IPython.core.inputtransformer2.html ).